styles/styles.css

```diff
@@ -90,170 +95,96 @@ header {
   height: var(--nav-height);
 }
 
-header .header,
-footer .footer {
-  visibility: hidden;
-}
-
-header .header[data-block-status="loaded"],
-footer .footer[data-block-status="loaded"] {
-  visibility: visible;
-}
-
 h1,
 h2,
 h3,
 h4,
-h5,
-h6 {
-  margin-top: 0.8em;
-  margin-bottom: 0.25em;
-  font-family: var(--heading-font-family);
-  font-weight: 600;
-  line-height: 1.25;
-  scroll-margin: 40px;
+h5 {
```

Comment:
Since all the Headers right now use the same family, weight, and style, they can go here, instead of on the individual definitions.

Reply:
Refactored as you recommended